tests/pipelines/pag/test_pag_pixart_sigma.py

```diff
@@ -127,7 +127,7 @@ def test_pag_disable_enable(self):
         out = pipe(**inputs).images[0, -3:, -3:, -1]
 
         # pag disabled with pag_scale=0.0
-        components["pag_applied_layers"] = [1]
+        components["pag_applied_layers"] = ["1"]
```

Comment:
Shouldn't this be "block.1"?

Reply:
Yes, my bad. Will update these. The intended usage is "blocks.{index}" or some regex combo of layers, but just strings like this work too (in this case) since the PAG logic is able to fulfil the 3 conditions mentioned. It will give unexpected behaviour if you have layers named like:

```
something.blocks.1.another_something.1.attn1
something.blocks.42.another_something.1.attn1
```

In this case, the intent was probably to only apply to `something.blocks.1` but PAG will get applied to both. But the general guide would be to follow `blocks.{}` convention because we use it consistently in naming our blocks. With regex, there are many ways to shoot yourself in the foot and so we can't really account for all cases if users don't want to follow the general guide.

Reply:
Yeah fair enough. Maybe we can update the documentation to be explicit about that? PAG is a very widely used feature, so, need to be rigorous. WDYT?